backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/comments/CommentEntryRepository.java

```diff
@@ -67,6 +84,8 @@ public void createCommentEntry(final @NonNull String characterData, @NonNull fin
 		chatEntry.setCharacterData(characterData);
 		chatEntry.setChatEntryType(X_CM_ChatEntry.CHATENTRYTYPE_NoteFlat);
 		InterfaceWrapperHelper.save(chatEntry);
+
+		referenceHasCommentsCache.put(tableRecordReference, true);
```

Comment:
better just invalidate the cache for that `tableRecordReference`.
keep in mind that ur method is executed in trx...

Reply:
done